### PR TITLE
esearch: add examples

### PR DESCRIPTION
**Pull request by koolape**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 17.9

### CODE REVIEW

File: pages/linux/esearch.md [hunk 1]
```diff
@@ -8,6 +8,14 @@
...
 
+- Search the protein database using a query and regexp:
+
+`esearch -db {{protein}} -query {{'Escherichia*'}}`
+
 - Search the nucleotide database for sequences whose metadata contain insulin and rodents:
 
 `esearch -db nuccore -query "{{insulin [PROT] AND rodents [ORGN]}}"`
+
+- Display help:
```
Comment (kbdharun) [inline pages/linux/esearch.md:19]:
```suggestion
- Display [h]elp:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a small suggestion.

File: pages/linux/esearch.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/esearch.md]: Are you sure? It isn't invalid, but seems a really simple example to highlights the argument in the description

Comment (koolape) [reply pages/linux/esearch.md]: the `-h` or `-help` option displays all command line options for esearch

Comment (kbdharun) [reply pages/linux/esearch.md]: > Are you sure? It isn't invalid, but seems a really simple example to highlights the argument in the description

It is valid. I usually don't make this suggestion for help command. But I found this in the docs `Each program has a ‑help command that prints detailed information about available arguments.` so I made the suggestion.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**